stellar/its-example.js

```diff
@@ -0,0 +1,138 @@
+'use strict';
+
+const { Contract, Address, nativeToScVal } = require('@stellar/stellar-sdk');
+const { Command } = require('commander');
+const { loadConfig, saveConfig } = require('../evm/utils');
+const { getWallet, broadcast, addBaseOptions, tokenToScVal, tokenMetadataToScVal } = require('./utils');
+const { addOptionsToCommands, getChainConfig } = require('../common');
+const { ethers } = require('hardhat');
+const {
+    utils: { arrayify, hexZeroPad },
+} = ethers;
+require('./cli-utils');
+
+async function deployToken(wallet, _, chain, contractConfig, args, options) {
```

Comment:
```suggestion
async function deployInterchainToken(wallet, _, chain, contractConfig, args, options) {
```
use the same name as the method for readability